skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
```

Comment:
@vighneshbirodkar yes, you are absolutely right.
My idea was to first implement it in Python and later check how 
it works with OpenCv trained data file and if it works, then I will for sure rewrite it
in Cython. It is needed because the numbers that I gave are for training and it has to be fast.
Also the evaluation phase has to be fast. But first I want to make sure that it just works.